ci/get-engine.ps1

```diff
@@ -2,7 +2,7 @@ param(
     # Note: this directory is outside the build directory and will not get automatically cleaned up from agents unless agents are restarted.
     [string] $engine_cache_directory = "$($pwd.drive.root)UnrealEngine-Cache",
     # Unreal path is a symlink to a specific Engine version located in Engine cache directory.
-    [string] $unreal_path = "$($gdk_home)\UnrealEngine",
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine",
```

Comment:
For places with paths like this (.parent.parent) etc, can we get a comment above with an example of what this string should look like to make it easier to maintain